src/screens/OrgSettings/OrgSettings.test.tsx

```diff
@@ -67,7 +68,7 @@ describe('Organisation Settings Page', () => {
     window.location.assign('/orglist');
 
     const { container } = render(
-      <MockedProvider addTypename={false} mocks={MOCKS}>
+      <MockedProvider addTypename={false} link={mocklink}>
```

Comment:
Please don't use mocklink

Reply:
Don't worry about the changes. I am developing a new function to rectify this error.

Reply:
What does this mean? Should the PR and issue be closed?